src/scss/base/_variables.scss

```diff
@@ -4,3 +4,8 @@ $blue-prim-color: #0290ff;
 $blue-secondary-color: #4386bf;
 $black-prim-color: #1d1919;
 $gray-color: #888;
+$gary-line: #e8e8e8;
+$p: 80%;
+$v: calc(((18 / 5) * $p - 90) * 1deg);
```

Comment:
After the application compiling in my local environment, this variable is frowned upon and cannot allow the project to run further. It only compiles after I comment it out. Below is the screenshot of the error 👇🏻 

![image](https://user-images.githubusercontent.com/4247740/156672917-440511d3-5541-4f72-9015-970c481f3f9b.png)

- [ ] Please kindly fix this error